OCCommunicationLib

```diff
@@ -1 +1 @@
-Subproject commit 374a2fc7eeb0343f3514d5777237bdd20d412c97
+Subproject commit f886baa1289986aacbce74fe1d8902590a9506e0
```

Comment:
The commit of the OCLibrary was updated to other previous, the correct one is 374a2fc7eeb0343f3514d5777237bdd20d412c97, right now master is tracked there.